packages/documentation/src/stories/components/switch/switch.stories.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import { Meta, Args, Story, StoryContext, ReactFramework } from '@storybook/react';
+import { useArgs } from '@storybook/client-api';
+import docsPage from './switch.docs.mdx';
+
+const VALIDATION_STATE_MAP: Record<string, undefined | boolean> = {
+  'null': undefined,
+  'is-valid': false,
+  'is-invalid': true
+};
+
+export default {
+  title: 'Components/Switch',
+  parameters: {
+    docs: {
+      page: docsPage
+    }
+  },
+  args: {
+    labelPosition: 'both',
+    labelBefore: 'Off',
+    labelAfter: 'On',
+    hiddenLabel: false,
+    checked: false,
+    disabled: false,
+    validation: 'null',
+    validFeedback: 'Ggranda sukceso!',
+    invalidFeedback: 'Eraro okazis!'
+  },
+  argTypes: {
+    labelPosition: {
+      name: 'Label Position',
+      description: 'The position of the components label.',
+      control: {
+        type: 'radio',
+        label: {
+          both: 'Both',
+          before: 'Before',
+          after: 'After'
+        }
+      },
+      options: [
+        'both',
+        'before',
+        'after'
+      ],
+      table: {
+        category: 'General'
+      }
+    },
+    labelBefore: {
+      name: 'Label (before)',
+      description: 'Describes the content/topic of the component.',
+      if: {
+        arg: 'labelPosition',
+        neq: 'after'
+      },
+      control: {
+        type: 'text'
+      },
+      table: {
+        category: 'General'
+      }
+    },
+    labelAfter: {
+      name: 'Label (after)',
+      description: 'Describes the content/topic of the component.',
+      if: {
+        arg: 'labelPosition',
+        neq: 'before'
+      },
+      control: {
+        type: 'text'
+      },
+      table: {
+        category: 'General'
+      }
+    },
+    hiddenLabel: {
+      name: 'Hidden Label',
+      description: '<p>Render the component with or without a visible label.</p><div className="alert alert-info alert-sm">There are accessibility issues with hidden labels.<br/>Please read our <a href="/?path=/story/foundations-accessibility--page#labels">labels accessibility guide</a>.</div>',
+      control: {
+        type: 'boolean'
+      },
+      table: {
+        category: 'General'
+      }
+    },
+    checked: {
+      name: 'Checked',
+      description: 'Defienes the checked state of the component.',
+      control: {
+        type: 'boolean'
+      },
+      table: {
+        category: 'States'
+      }
+    },
+    disabled: {
+      name: 'Disabled',
+      description: '<p>When set to `true`, disables the component\'s functionality and places it in a disabled state.</p><div className="alert alert-info alert-sm">There are accessibility issues with the disabled state.<br/>Please read our <a href="/?path=/docs/foundations-accessibility--page#disabled-state">disabled state accessibility guide</a>.</div>',
```

Comment:
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/1659006/207112655-8e5418cd-537b-475b-95c3-24051b2053c4.png">

The p tag seems to be styled with Post styles where a regular description is not. This can also be a separate ticket, but I think we should uniformly style all text the post way.

Reply:
Because of another review from @alizedebray, I removed the p-tag entirely.
As you suggested, we can fix the styling of the ArgsTable in seperate ticket (see #880).